client/src/modules/CourseStatistics/components/TaskPerformanceCard/TaskPerformanceCard.tsx

```diff
@@ -57,8 +58,41 @@ export const TaskPerformanceCard = ({ tasks }: Props) => {
   );
 };
 
-function getChartConfig() {
+function getPerformanceDescriptionByType(type: string) {
```

Comment:
Minor: I would propose to create some enum that will contain all cases and use it instead of just `string`